pipelines/jobs/run-synapse-data-validation-tests.yaml

```diff
@@ -0,0 +1,26 @@
+parameters:
+  agentPool: ''
+  env: ''
+
+##
+# Run data validation tests
+##
+jobs:
+- job: RunDataValidationTestsJob
+  pool: ${{ parameters.agentPool }}
+  timeoutInMinutes: 0 # Max timeout
+  steps:
+  - checkout: self
+    clean: true
+  - template: ${{variables['System.DefaultWorkingDirectory']}}/pipelines/steps/azure-login.yaml
+  - script: |
+      export PATH="$PATH:/home/AzDevOps/.local/bin/"
+      sudo apt-get install unixodbc-dev
```

Comment:
Is it possible to pre-install the dependencies via agent image ? To avoid delays and security issue.
Can be risky to use sudo and curl in the shared agents